ports/minimp3/portfile.cmake

```
@@ -1,13 +1,13 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO lieff/minimp3
-    REF 95864e8e0d3b34402a49ae9af6c66f7e98c13c35 #committed on Nov 27
-    SHA512 6e5364a83e882b54fd1eb5ba35ec7c7179b5b5c0ceb2c658615a2306ae0c28252ca8591ec6b515483c6ff0ed608db7eb73fba3d201a20ad4a85ce7b3a091a695
+    REF afb604c06bc8beb145fecd42c0ceb5bda8795144 # committed on 2021-11-30
+    SHA512 633da0b20982f6f22c87d872c69626b2939ffb4519339cd0c090d7538308007cf633c07af57020cd2332a75c6e7b9bf3ebd5bda1af59dc96a4f0e85ce1b3f751
     HEAD_REF master
 )
 
 file(COPY ${SOURCE_PATH}/minimp3.h DESTINATION ${CURRENT_PACKAGES_DIR}/include/${PORT})
 file(COPY ${SOURCE_PATH}/minimp3_ex.h DESTINATION ${CURRENT_PACKAGES_DIR}/include/${PORT})

```

```suggestion
file(COPY ${SOURCE_PATH}/minimp3.h DESTINATION ${CURRENT_PACKAGES_DIR}/include/${PORT})
file(COPY ${SOURCE_PATH}/minimp3_ex.h DESTINATION ${CURRENT_PACKAGES_DIR}/include/${PORT})

```
For port updates, I expect you to add double quotes to any absolute paths. Of course, it's a matter of nitpicking. You can choose not to change, please let me know what you think. :)

---

I understand the significance of quotes, I just tried to keep the changes minimal.